### DISCUSSION TITLE
Get visible text on the java window

### DISCUSSION THREAD

**Discussion by automonster** [gaozhao1989/pyjab#66] [General]
I'm looking for a function to get the visible text on the window. My code looks like below:
jabdriver = JABDriver("Java Control Panel", timeout=60)
text_fields = jabdriver.find_elements_by_role("text")
text_field = jabdriver.find_element_by_object_depth(11)

Now I want to get the visible text on the window "The Java Update mechanism ensures you have the most updated version of the Java Platform. The options below let you control how updates are obtained and applied."

@gaozhao1989 Kindly help me out.

![image](https://user-images.githubusercontent.com/117296753/199531777-ee9f7b88-885a-4e9e-af78-ad70826b3519.png)

**Comment by automonster** [comment]
figured out that I could do it by text_field.get_element_information()['text'] :)

**Reply by gaozhao1989** [reply]
You can use xxx_element.text get text from JABElement after you find it.

**Reply by automonster** [reply]
Yes. It worked and looks very simple. Thank you @gaozhao1989
